website/static/js/pages/register_1-page.js

```diff
@@ -4,16 +4,22 @@ var ko = require('knockout');
 var bootbox = require('bootbox');
 var $osf = require('js/osfHelpers');
 
-var MetaData = require('../metadata_1.js');
+var MetaData = require('js/metadata_1.js');
 var ctx = window.contextVars;
 /**
     * Unblock UI and display error modal
     */
-function registration_failed() {
+function registrationFailed() {
     $osf.unblock();
     bootbox.alert('Registration failed');
 }
 
+function postRegisterNode() {
+    $('#registration_template').children().remove();
+    var msg = "Your registration request was submitted successfully. Files are being copied to the newly created registration, and you will recieve an email notification containing a link to the registration when the copying is finished.";
```

Comment:
Do we want to specifically call this archiving as its a bit different from copying?